### PR TITLE
[Office] Add correct key name for runtime logging

### PR DESCRIPTION
**Pull request by davidchesnut**
Added correct name of "RuntimeLogging" for creating registry key.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3fff08b](https://github.com/OfficeDev/office-js-docs-pr/commits/3fff08b6924b72eb8a8b7449316ee23ca0423c2c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-logfix/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910252147505336-1392%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=9tVb0Y5be7saljrv1XORLAcw%2BF4tq%2F%2F941CG4T%2BF5aA%3D&st=2019-10-25T21%3A44%3A09Z&se=2019-11-25T21%3A49%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c42ca5d](https://github.com/OfficeDev/office-js-docs-pr/commits/c42ca5db69410f5d97e88cdd4e089447335e4cb4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-logfix/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910252149462023-1392%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=8WUj0N2a1gVNZobEFVNz717z%2FWsFMYRjz4a%2B2NHoe1A%3D&st=2019-10-25T21%3A46%3A06Z&se=2019-11-25T21%3A51%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/troubleshoot-manifest.md [hunk 1]
```diff
@@ -70,9 +70,9 @@ You can use runtime logging to debug your add-in's manifest as well as several i
 2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Developer\`. 
 
     > [!NOTE]
-    > If the `Developer` key (folder) does not already exist under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\`, complete the following steps to create it: 
-	> 1. Right-click the **WEF** key (folder) and select **New** > **Key**.
-	> 2. Name the new key **Developer**.
+    > If the `Developer` key (folder) does not already exist under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Developer`, complete the following steps to create it:
```
Comment (kbrandl) [inline docs/testing/troubleshoot-manifest.md]:
```suggestion
    > If the `RuntimeLogging` key (folder) does not already exist under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Developer`, complete the following steps to create it: 
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[afa4446](https://github.com/OfficeDev/office-js-docs-pr/commits/afa4446b4bb7bd9f838ad83011bd38daed572105)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-logfix/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910252211455100-1392%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=f441WomJtKrJpQED8weJXY76vkmcbUrhT%2FdljqVkwyA%3D&st=2019-10-25T22%3A08%3A06Z&se=2019-11-25T22%3A13%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a2cdc51](https://github.com/OfficeDev/office-js-docs-pr/commits/a2cdc5194dcfe5bcfd057e3a88df1029d1cde4e2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-logfix/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910252212321498-1392%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=UZYWJOrQHxKSOkE%2FwdC3vMiRBHGJFiKyQQuJiCdR7pQ%3D&st=2019-10-25T22%3A09%3A20Z&se=2019-11-25T22%3A14%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[66ca24d](https://github.com/OfficeDev/office-js-docs-pr/commits/66ca24d46ba06f3ba3770b8e23f883fd57810305)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-logfix/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910252215248802-1392%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=dWNbpt7%2B8dUHdXZWYG2K1%2BkWAmNh%2BYfpxXDboEkbnHQ%3D&st=2019-10-25T22%3A11%3A32Z&se=2019-11-25T22%3A16%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[72b73c2](https://github.com/OfficeDev/office-js-docs-pr/commits/72b73c22844b71576566890b0e5aa7181cd63bfe)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-logfix/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910252216029773-1392%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Mud0HMT2Hu3t3ORmcDKEy%2BlNShQVhOOWiw%2B986zkg%2B4%3D&st=2019-10-25T22%3A12%3A47Z&se=2019-11-25T22%3A17%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/troubleshoot-manifest.md [hunk 2]
```diff
@@ -67,14 +67,14 @@ You can use runtime logging to debug your add-in's manifest as well as several i
 
 1. Make sure that you are running Office 2016 desktop build **16.0.7019** or later. 
 
-2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Developer\`. 
+2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef\Developer\`.
```
Comment (kbrandl) [inline docs/testing/troubleshoot-manifest.md]: Seems like "WEF" should remain capitalized -- it's showing like that for me, anyway:

image

File: docs/testing/troubleshoot-manifest.md [hunk 3]
```diff
@@ -67,14 +67,14 @@ You can use runtime logging to debug your add-in's manifest as well as several i
 
 1. Make sure that you are running Office 2016 desktop build **16.0.7019** or later. 
 
-2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Developer\`. 
+2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef\Developer\`. 
 
     > [!NOTE]
-    > If the `Developer` key (folder) does not already exist under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\`, complete the following steps to create it: 
-	> 1. Right-click the **WEF** key (folder) and select **New** > **Key**.
+    > If the `Developer` key (folder) does not already exist under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef`, complete the following steps to create it:
```
Comment (kbrandl) [inline docs/testing/troubleshoot-manifest.md]: per previous comment, should remain capitalized:   WEF

File: docs/testing/troubleshoot-manifest.md [hunk 4]
```diff
@@ -67,14 +67,14 @@ You can use runtime logging to debug your add-in's manifest as well as several i
...
 1. Make sure that you are running Office 2016 desktop build **16.0.7019** or later. 
 
-2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\Developer\`. 
+2. Add the `RuntimeLogging` registry key under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef\Developer\`. 
 
     > [!NOTE]
-    > If the `Developer` key (folder) does not already exist under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\WEF\`, complete the following steps to create it: 
-	> 1. Right-click the **WEF** key (folder) and select **New** > **Key**.
+    > If the `Developer` key (folder) does not already exist under `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\16.0\Wef`, complete the following steps to create it: 
+	> 1. Right-click the **Wef** key (folder) and select **New** > **Key**.
```
Comment (kbrandl) [inline docs/testing/troubleshoot-manifest.md]: per previous comment, should remain capitalized: WEF

**Comment by o365devx** [comment]
Docs Build status updates of commit _[fbca909](https://github.com/OfficeDev/office-js-docs-pr/commits/fbca9091e556d66354361c8dd44e8530408a05b4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-logfix/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910312038422997-1392%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=srSh4ksm1FDHpOq%2BtxNaHo3PGV58TkM5q6CBJKzypmU%3D&st=2019-10-31T20%3A34%3A44Z&se=2019-12-01T20%3A39%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[51e5886](https://github.com/OfficeDev/office-js-docs-pr/commits/51e58867715d98991f724fb18ded98958202e733)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/troubleshoot-manifest.md](https://github.com/OfficeDev/office-js-docs-pr/blob/dc-logfix/docs/testing/troubleshoot-manifest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201910312039289002-1392%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ARy7mRk5nhTK6FZyPBgZUe6x4UtYc6ql5E81ak7%2Fr3o%3D&st=2019-10-31T20%3A35%3A53Z&se=2019-12-01T20%3A40%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**